src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/__tests__/3.1.0.test.ts

```diff
@@ -265,35 +331,185 @@ describe('USD Tiered STO 3.1.0', () => {
       when(mockedWrapper.getAvailableAddressesAsync()).thenResolve([expectedOwnerResult]);
 
       const mockedParams = {
+        customOracleAddresses: [
+          '0x1111111111111111111111111111111111111111',
+          '0x2221111111111111111111111111111111111111',
+        ],
+        denominatedCurrencySymbol: 'DAI',
         txData: {},
```

Comment:
I’m a bit confused why it’s dai here and usdt below

Reply:
the method has different logic if the user changes the denominated currency symbol and it's different from the old one. In this case I would like to change USDT symbol to DAI symbol.

Reply:
Denominated currency symbol is supposed to be `USD` or `CAD` or `EUR`, etc, not tokens.

Reply:
Yes, you're right! I thought on that and forget to change it to be more consistent